modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -21,6 +21,8 @@ hosted control plane:: An {product-title} control plane that runs on the managem
 
 hosting cluster:: See _management cluster_.
```

Comment:
Good question. I'd prefer to keep it in case someone is searching specifically for the term "hosting cluster", as that's used more commonly in the ACM docs.